.github/workflows/ci.yml

```diff
@@ -127,7 +126,8 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          pip install --upgrade pip setuptools py
+          pip install "pip<20.3"
+          pip install "setuptools==44.0.0" # for py27
```

Comment:
The `setuptools` bit should also go to `maint-0.7`, but we can do it as another PR there.